build/workflow/pipeline.yml

```diff
@@ -55,12 +55,10 @@ jobs:
       Samples:
         ApplicationPlatform: Samples
         ArtifactName: Samples
-        DisableNet6MobileTargets: true
```

Comment:
Why do you remove those? That is unless samples also now build with net6, but I don't think they do.